### PR TITLE
feat(harmonised footer): add component - INNO-1820

### PR DESCRIPTION
**Pull request by emeryro**
# PR description

specs: https://webgate.ec.europa.eu/CITnet/confluence/display/NEXTEUROPA/Harmonised+Footer+-+Design+Specifications

playground: 
https://deploy-preview-1318--europa-component-library.netlify.com/playground/ec/?path=/story/components-footers-harmonised--group-1
https://deploy-preview-1318--europa-component-library.netlify.com/playground/ec/?path=/story/components-footers-harmonised--group-2

### CODE REVIEW

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 9fa892580e534417072fb5a351c5320ba469b1eb

https://deploy-preview-1318--europa-component-library.netlify.com

**Comment by HAL-Patch-INNO** [comment]
Deploy preview for *europa-component-library* ready!

Built with commit 568cc1b79f2ea026bf58dc930b3769b17b4261eb

https://deploy-preview-1318--europa-component-library.netlify.com

File: .size-limit.json [hunk 1]
```diff
@@ -20,7 +20,7 @@
   {
     "path": "dist/packages/ec-preset-legacy/styles/ecl-ec-preset-legacy.css",
     "webpack": false,
-    "limit": "21 KB"
+    "limit": "22 KB"
```
Comment (yhuard) [inline .size-limit.json:23]: :'(

File: src/systems/ec/implementations/vanilla/packages/ec-component-footer-core/ec-component-footer-core.scss [hunk 2]
```diff
@@ -9,7 +9,6 @@
 // Check if overridden dependencies are already loaded, if needed
 @include check-imports(('ec-layout-grid', 'ec-component-link'));
 
-/* stylelint-disable value-no-vendor-prefix, property-no-vendor-prefix */
```
Comment (yhuard) [inline src/systems/ec/implementations/vanilla/packages/ec-component-footer-core/ec-component-footer-core.scss:12]: :)

**Review by yhuard — APPROVED** [review]

**Pull request merged**